e

```diff
@@ -0,0 +1,2191 @@
+Author: Lei Qiao <qiaol618@gmail.com>
+Date:   Sun May 17 12:18:54 2015 -0500
+
+    suppress warning unused function 'get_element'
+
+[33mcommit 32d3366d8486c6424c59db1b43627659dc0a38f3[m
+Merge: d0fcb6a 2e515f1
+Author: Luca Heltai <luca.heltai@sissa.it>
+Date:   Sat May 16 21:05:21 2015 +0200
+
+    Merge pull request #931 from QiaoLei-88/step33_revise
+    
+    Step33 revise
+
+[33mcommit 28f9de606d3f31ef3d2eeb7a22d19174b2776080[m
+Author: Luca Heltai <luca.heltai@sissa.it>
+Date:   Sat May 16 20:53:35 2015 +0200
+
+    Restructured test for MappingFEField.
+
+[33mcommit fea340c2ae021234daf0e3f99c5086d5c41c535b[m
+Author: Luca Heltai <luca.heltai@sissa.it>
+Date:   Fri May 15 22:36:22 2015 +0200
+
+    Fixed names and typedefs.
+
+[33mcommit bc307f50672c50062e8a581ea75d1284a5f75380[m
+Author: nicola giuliani <nicolagiuliani@dhcp-129-199.sissa.it>
+Date:   Fri May 15 17:59:09 2015 +0200
+
+    added the failing test for mappingfe
+
+[33mcommit d0fcb6adce06edb16d60ab7bb5ddfeb4ee2c562c[m
+Merge: 0561dfe 08e3b26
+Author: Luca Heltai <luca.heltai@sissa.it>
+Date:   Sat May 16 18:34:02 2015 +0200
+
+    Merge pull request #937 from bangerth/change-function-name-in-step-49
+    
+    Change mesh_info() to print_mesh_info().
+
+[33mcommit 08e3b26cf445928f9f2b74f0d698b6f7eaa7c9ca[m
+Author: Wolfgang Bangerth <bangerth@math.tamu.edu>
+Date:   Sat May 16 06:10:02 2015 -0500
+
+    Change mesh_info() to print_mesh_info().
+    
+    Our function names almost always have a verb in it that indicates
+    what the function does. This one did not. Change it.
+
+[33mcommit 0561dfe88e9352a5eb310f6c8bb2e4bbf011f139[m
+Merge: e0817eb 4762942
+Author: Wolfgang Bangerth <bangerth@math.tamu.edu>
+Date:   Fri May 15 22:48:19 2015 -0500
+
+    Merge pull request #936 from luca-heltai/derivative-form-number
+    
+    Added template Number to DerivativeForms.
+
+[33mcommit 47629428e77465edde9eba8565f7e793476531a8[m
+Author: Luca Heltai <luca.heltai@sissa.it>
+Date:   Sat May 16 00:55:55 2015 +0200
+
+    Added template Number to DerivativeForms.
+
+[33mcommit e0817eb7f6005772c635778558a60468ad8c6471[m
+Merge: 3d24a2a 6f169d4
+Author: Luca Heltai <luca.heltai@sissa.it>
+Date:   Fri May 15 00:19:19 2015 +0200
+
+    Merge pull request #933 from andreamola/opencascade_bug_fix
+    
+    Fixed bug in opencascade/boundary_lib.cc
+
+[33mcommit 6f169d478de9b25b6c7ab2cb835899cbcb0288c9[m
+Author: Luca Heltai <luca.heltai@sissa.it>
+Date:   Fri May 15 00:17:19 2015 +0200
+
+    Made dist const.
+
+[33mcommit b938c857fcea41c65a825e5a71ad1eab305740fd[m
+Author: Luca Heltai <luca.heltai@sissa.it>
+Date:   Fri May 15 00:15:03 2015 +0200
+
+    Use void instead of bogus operation.
+
+[33mcommit 3d24a2accf25a1a251210b1a2f487b48f410fc70[m
+Merge: a921d0f cc15d35
+Author: Wolfgang Bangerth <bangerth@math.tamu.edu>
+Date:   Thu May 14 15:55:05 2015 -0500
+
+    Merge pull request #924 from gpitton/q_chebyshe
```

Comment:
What is this file good for?

Can you please remove it and squash the changes so that we do not end up with 2000 lines of code more in the repository?